### DISCUSSION TITLE
Upcoming release v2024.1 - Thu 09/05/2024

### DISCUSSION THREAD

**Discussion by jenriquesoriano** [INSPIRE-MIF/helpdesk-validator#1073] [Announcements]
Dear Community,

we are writing to share an update regarding the upcoming release of the INSPIRE Reference Validator (v2024.1).

This release will introduce various changes, including an architectural and technological redesign aimed at simplifying the operation of the INSPIRE Reference Validator.

We are currently in the final stages of testing to ensure a smooth user experience.

We will keep you informed of the progress and will provide a revised release date as soon as it becomes available, likely within the upcoming week.

In the meantime, if you have any questions or concerns, please don't hesitate to reach out to us.

Thank you.

Regards,

**Comment by DanielMartinPerezdeLeon** [comment]
Dear community,

regarding the v2024.1 release of the INSPIRE Reference Validator, we will be conducting a series of operations on the production instance to ensure a smooth deployment of the v2024.1 release planned for next week. 
These operations will take place in the coming days between 8:00 AM and 10:00 AM Central European Time (CET). 

During this period, you may experience some unavailability or disruptions in the operation of the INSPIRE Reference Validator [Production instance](https://inspire.ec.europa.eu/validator/home/index.html).

In case you encounter any issues, we recommend using the [Staging instance](https://staging-inspire-validator.eu-west-1.elasticbeanstalk.com/etf-webapp/home/index.html) as an alternative. The Staging instance is designed for testing and validation purposes, and it should remain unaffected during the maintenance window.

Thank you for your understanding, and if you have any further questions or concerns, feel free to reach out to us.

Regards,

**Comment by jenriquesoriano** [comment]
Dear INSPIRE users,

we are in the process to deploy a new release today Thursday 09/05/2024 during the next hours.
This release, additionally to ETSs updates, is mainly focused on deploying a new version with a technological and architectural update that may potentially facilitate the maintenance of the platform, especially for those of you using it on your own premises.

We will keep you informed with the release status and the [Production instance](https://inspire.ec.europa.eu/validator/home/index.html) availability during the process.
In the meantime, if you experience issues in the [Production instance](https://inspire.ec.europa.eu/validator/home/index.html), we kindly invite you to use the [Staging instance](https://staging-inspire-validator.eu-west-1.elasticbeanstalk.com/etf-webapp/home/index.html) instead.

Please take into account that the existing Test Reports might be deleted to deploy this fresh [Production instance](https://inspire.ec.europa.eu/validator/home/index.html), so we encourage you to save any Test Run that you may want to preserve.

We will keep you posted with the service status.

Kind regards,

**Comment by jenriquesoriano** [comment]
Dear INSPIRE users,

just inform that the v2024.1 release has been deployed in the [Production instance](https://inspire.ec.europa.eu/validator/home/index.html).
For further details, please visit the [release notes](https://github.com/INSPIRE-MIF/helpdesk-validator/releases/tag/v2024.1).

Kind regards,
